src/Evolution/Executables/Cce/CharacteristicExtract.hpp

```diff
@@ -127,8 +127,7 @@ struct EvolutionMetavars {
 };
 
 static const std::vector<void (*)()> charm_init_node_funcs{
-    &setup_error_handling,
-    &disable_openblas_multithreading,
+    &setup_error_handling, &disable_openblas_multithreading,
```

Comment:
How do I remove this change from a history commit? @moxcodes

Reply:
The way I would do it is to open an interactive rebase with `git rebase -i [parent commit]` and mark the commit to alter with `edit` (instead of `pick`), and then you can `git checkout -p [previous commit hash] [filename]`, it will ask you for each hunk of the commit whether you want to checkout the previous commit's changes. You can just answer 'n' to most of them, and 'y' to the one for this hunk. Once you finish with that, you can `git commit --amend` to put the alterations in the edited commit, then `git rebase --continue` to get back to your branch head.